.github/workflows/ci-mlir.yml

```diff
@@ -32,10 +32,13 @@ jobs:
         python-version: ${{ matrix.python-version }}
 
     - name: Clang Setup
-      uses: egor-tensin/setup-clang@v1
+      uses: KyleMayes/install-llvm-action@v1
+      with:
+        version: "15.0"
+        ubuntu-version: 18.04
 
     - name: Ninja Setup
-      uses: llvm/actions/install-ninja@main
+      uses: lukka/get-cmake@latest
```

Comment:
How often is this updated?
Hopefully, we would like to only have this updated at most every month, since this will trigger a new recompilation, right?

Reply:
Moved to a specific commit instead of the branch.